frontend/invites/new_invite_page.html

```diff
@@ -2,6 +2,9 @@
   <div layout="column" layout-align="center center" layout-fill flex>
     <div layout="row" layout-xs="column" layout-align="center center">
       <div flex-sm="60" flex-gt-sm="40" layout-align="center center">
+        <div ng-show="newInviteCtrl.loading" layout="row" layout-align="center center">
+          <md-progress-circular md-mode="indeterminate" style="width: 20%"></md-progress-circular>
```

Comment:
That width in percentage it will be extremely big in large screens and really small in smartphones screens. Just don't change the size.